bind/gencffi.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2017 The go-python Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package bind
+
+import (
+	"go/token"
+)
+
+const (
+	builderPreamble = `import os
+
+from sys import argv
+from cffi import FFI
+
+os.chdir(os.path.dirname(os.path.abspath(__file__)))
+ffibuilder = FFI()
+ffibuilder.set_source(
+    'gopy.%[1]s',
```

Comment:
@sbinet 
This is just related to CFFI library.
The user will import as `import XXXX` not `import gopy.XXXX`